Modules/Synchronization/src/Synchronization.Application/IntegrationEvents/Incoming/IdentityDeletionProcessStatusChanged/IdentityDeletionProcessStatusChangedIntegrationEvent.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.EventBus.Events;
+
+namespace Backbone.Modules.Synchronization.Application.IntegrationEvents.Incoming.IdentityDeletionProcessStatusChanged;
+public class IdentityDeletionProcessStatusChangedIntegrationEvent : IntegrationEvent
+{
+    public IdentityDeletionProcessStatusChangedIntegrationEvent(string identityAddress, string deletionProcessId)
+        : base($"{identityAddress}/IdentityDeletionProcessStatusChanged/{deletionProcessId}")
+    {
+        Address = identityAddress;
+        DeletionProcessId = deletionProcessId;
+    }
+
+    public string Address { get; }
+    public object DeletionProcessId { get; }
```

Comment:
Why object?

Reply:
Doing things in a TDD way and using VS to create methods, props... automatically, I should have changed all of these during the actual implementation. No excuse how I missed this one...

Reply:
That's what I expected. And to be honest: this has been happening to me as well. So no worries. ;) I'm happy to see that you're actually doing TDD.